resources/js/store/auth/index.js

```diff
@@ -0,0 +1,66 @@
+const auth = {
+  namespaced: true,
+  state() {
+    return { errors: {} };
+  },
+  mutations: {
+    login(state, payload) {
+      //   console.log(payload);
+      if (payload.status == 200) {
+        window.location.href = "/task";
```

Comment:
Again, the mutation job is just to change the data of the `state` and nothing else. we can't write code to redirect in the mutation. That has to be done in the action itself.